src/fit.py

```diff
@@ -1,23 +1,48 @@
 """Contains the necessary tools to fit a dataset."""
 
+import os.path
+import typing as tp
+
 import matplotlib.pyplot as plt
 import numpy as np
 from scipy.optimize import curve_fit
 
 # We need the value of the earth gravity to determine the initial velocity and initial angle
 from generate import EARTH_GRAVITY
 
+REPO_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+DATA_DIR = os.path.join(REPO_DIR, "data")
+PLOT_DIR = os.path.join(REPO_DIR, "plots")
```

Comment:
[`pathlib`](https://docs.python.org/3/library/pathlib.html#module-pathlib) is the high-level interface to path manipulation and should be preferred nowadays I think. Also `path / "data"` feels much more natural :upside_down_face: